diesel_cli/src/cli.rs

```diff
@@ -124,32 +124,60 @@ pub fn build_cli() -> App<'static, 'static> {
                 .short("o")
                 .long("only-tables")
                 .help("Only include tables from table-name")
+                .conflicts_with("only-table-regexes")
```

Comment:
The next release will be 2.0 so we can make a breaking change here. Therefore I would prefer to just introduce this functionality under the old flags.

cc @diesel-rs/core  as information.

Reply:
What does the old flags mean?

Reply:
I'm thinking of just replacing the old flag (`only-tables`) with the regex filtering.